test/pylib/manager_client.py

```diff
@@ -351,15 +355,16 @@ async def servers_add(self, servers_num: int = 1,
                           start: bool = True,
                           seeds: Optional[List[IPAddress]] = None,
                           driver_connect_opts: dict[str, Any] = {},
-                          expected_error: Optional[str] = None) -> List[ServerInfo]:
+                          expected_error: Optional[str] = None,
+                          server_encryption: str = "none") -> List[ServerInfo]:
```

Comment:
do we actually want to pass the string "none" here by default (i.e. setting the encryption to "none" explicitly) instead of passing `None` and omit it completely?

Reply:
Not setting "none" explicitly would be more complicate because then the whole encryption section in the confing needs to be conditionally committed. It was simpler to include it unconditionally and  always set encryption type.